### ISSUE TITLE
Outdated dependency:  jacoco-maven-plugin (0.8.1 instead of 0.8.0)

### ISSUE THREAD

**Issue by dzc34** [Asqatasun/Asqatasun#220]
## Observed behaviour
Outdated dependency: jacoco-maven-plugin

## Expected behaviour
- upgrade **jacoco-maven-plugin**  to  0.8.1	  instead of  0.8.0
   - https://github.com/jacoco/jacoco
   - https://github.com/jacoco/jacoco/releases/tag/v0.8.1
  - http://www.eclemma.org/jacoco/trunk/doc/changes.html

## File
[pom.xml](https://github.com/Asqatasun/Asqatasun/blob/develop/pom.xml#L94)

### How to test it?
```bash
# pre-requisite: 
# - git, maven
# - fork  the repository on Github
git clone https://github.com/<user>/Asqatasun.git
     # instead of:   git clone https://github.com/Asqatasun/Asqatasun.git
cd Asqatasun
git checkout develop
git checkout -b fix220

./docker/build_and_run-with-docker.sh --skip-build-test --source-dir $(pwd) --docker-dir docker/SNAPSHOT-local
    # report: */target/site/jacoco

# or 
mvn clean install 
    # report: */target/site/jacoco

# commit / push  + PR (pull request on Github)
git add .
git commit -m "Fixed #220  - Upgraded  jacoco-maven-plugin (0.8.1 instead of 0.8.0)"
git push origin fix220
```

**Comment by Barogthor** [comment]
I'll take it

**Comment by dzc34** [comment]
fixed by 000bb6641
